ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -96,7 +96,7 @@ module KDC : sig
   val server : t -> string
   (** IP address *)
 
-  val port : t -> int
+  val _port : t -> int
```

Comment:
why?

Reply:
It's a convention to show it's unused

Reply:
Indeed. Otherwise the compiler will complain. It also highlights that we don't pass the port to the Samba binaries.